examples/SFVehicles/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:2.2.0-alpha1'
```

Comment:
I'd use the last stable version here, these `-alpha` and `-beta` versions have an expiration date so in ~4 weeks when someone tries to import this Android Studio will give them some really cryptic error and fail.